cmd/thanos/receive.go

```diff
@@ -194,6 +194,18 @@ func runReceive(
 		return errors.Wrap(err, "parse relabel configuration")
 	}
 
+	var limitsConfig *receive.RootLimitsConfig
+	if conf.limitsConfig != nil {
+		limitsContentYaml, err := conf.limitsConfig.Content()
```

Comment:
As @fpetkovski mentioned, having some `/~/reload` endpoint like we have in Thanos Ruler, to reload such configuration would be amazing, as adding some tenant limits now, would mean redeploying.

Reply:
Yup, leaving this for another PR to avoid this one getting too big.